source/js/buyers-guide/product-analytics.js

```diff
@@ -0,0 +1,92 @@
+import ReactGA from 'react-ga';
+
+function getElementGAInformation(pageTitle, productName) {
+  return {
+    'product-live-chat`': {
+      category: `product`,
+      action: `customer support link tap`,
+      label: `support link for ${productName}`
+    },
+    'donate-button': {
+      category: `site`,
+      action: `donate tap`,
+      label: `${pageTitle} donate header`
+    },
+    'nav-social-button-fb': {
+      category: `site`,
+      action: `share tap`,
+      label: `share site to Facebook`,
+      transport: `beacon`
+    },
+    'nav-social-button-twitter': {
+      category: `site`,
+      action: `share tap`,
+      label: `share site to Twitter`,
+      transport: `beacon`
+    },
+    'nav-social-button-link': {
+      category: `site`,
+      action: `copy link tap`,
+      label: `copy link to site `
+    },
+    'nav-social-button-email': {
+      category: `site`,
+      action: `share tap`,
+      label: `share site to Email`,
+      transport: `beacon`
+    },
+    'product-copy-link-button': {
+      category: `product`,
+      action: `copy link tap`,
+      label: `copy link ${productName}`
+    },
+    'creep-vote': {
+      category: `product`,
+      action: `opinion submitted`,
+      label: `opinion on ${productName}`
+    },
+    'product-social-button-fb': {
```

Comment:
maybe these are todos for later - we need to assign elements with the following ids still `product-social-button-fb`, `product-social-button-twitter`, and `product-social-button-email`